doc/lua_api.txt

```diff
@@ -4707,9 +4712,10 @@ Call these functions only at load time!
     * `damage`: Number that represents the damage calculated by the engine
     * should return `true` to prevent the default damage mechanism
 * `minetest.register_on_rightclickplayer(function(player, clicker))`
-    * Called when a player is right-clicked
-    * `player`: ObjectRef - Player that was right-clicked
-    * `clicker`: ObjectRef - Object that right-clicked, may or may not be a player
+    * Called when the 'place/use' key was used while pointing a player
+      (not neccessarily an actual rightclick)
```

Comment:
"(not neccessarily an actual rightclick)" seems superfluous because `place/use` basically *triggers a rightclick*, except re-mapped to another key.